wcbench.sh

```diff
@@ -27,13 +27,13 @@ KARAF_SHELL_PORT=8101  # Port that the Karaf shell listens on
 CONTROLLER="OpenDaylight"  # Currently only support ODL
 CONTROLLER_IP="localhost"  # Change this to remote IP if running on two systems
 CONTROLLER_PORT=6633  # Default port for OpenDaylight
-SSH_HOSTNAME="cbenchc"  # You'll need to update this to reflect ~/.ssh/config
+SSH_HOSTNAME="localhost"  # You'll need to update this to reflect ~/.ssh/config
 
 # Paths used in this script
 BASE_DIR=$HOME  # Directory that code and such is dropped into
 OF_DIR=$BASE_DIR/openflow  # Directory that contains OpenFlow code
 OFLOPS_DIR=$BASE_DIR/oflops  # Directory that contains oflops repo
-ODL_DIR=$BASE_DIR/distribution-karaf-0.2.1-Helium-SR1  # Directory with ODL code
+ODL_DIR=$BASE_DIR/distribution-karaf-0.2.1-Helium-SR1  # Directory with ODL code Helium version SR3
```

Comment:
Since we're not actually using SR3, very much don't want to add wrong docs. I raised issue #67 to track the SR3 update.